mm2src/coins/eth.rs

```diff
@@ -5891,11 +5891,24 @@ impl MmCoin for EthCoin {
         dex_fee_amount: DexFee,
         stage: FeeApproxStage,
     ) -> TradePreimageResult<TradeFee> {
-        let dex_fee_amount = wei_from_big_decimal(&dex_fee_amount.fee_amount().into(), self.decimals)?;
+        let fee_coin = match &self.coin_type {
+            EthCoinType::Eth => self.ticker.to_owned(),
+            EthCoinType::Erc20 { platform, .. } => platform.to_owned(),
+            EthCoinType::Nft { .. } => return MmError::err(TradePreimageError::NftProtocolNotSupported),
+        };
```

Comment:
You can use `platform_ticker` instead. I am not sure if you intentionally drop the nft support, but if we want to do that we need to add a if check before using `platform_ticker`.